types/esbuild-copy-static-files/index.d.ts

```diff
@@ -0,0 +1,21 @@
+// Type definitions for esbuild-copy-static-files 0.1
+// Project: https://github.com/nickjj/esbuild-copy-static-files
+// Definitions by: David Enke <https://github.com/davidenke>
+//                 Tom Kopp <https://github.com/TomKopp>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/*~ You can declare types that are available via importing the module */
+export interface CopyStaticFilesOptions {
+    src: string;
+    dest: string;
+    filter: (src: string, dest: string) => boolean;
+
+    dereference: boolean;
+    errorOnExist: boolean;
+    force: boolean;
+    preserveTimestamps: boolean;
+    recursive: boolean;
+}
+
+declare function copyStaticFiles(options?: Partial<CopyStaticFilesOptions>): (build: any) => void;
```

Comment:
does it return a void or a promise?
https://github.com/evanw/esbuild/blob/main/lib/shared/types.ts#L271-L274

Reply:
scratch that one, this is an union, could work with void only

Reply:
But you're right, the return type is wrong anyway. It returns a plugin instance. I aligned it.